### PR TITLE
Keep negotiated codec parameters in Downtrack.Bind

### PR DESCRIPTION
**Pull request by cnderrauber**
the context's codec parameters will be set to the binded codec after Bind

### CODE REVIEW

File: pkg/sfu/downtrack.go [hunk 1]
```diff
@@ -459,6 +463,17 @@ func (d *DownTrack) Bind(t webrtc.TrackLocalContext) (webrtc.RTPCodecParameters,
 		return codec, nil
 	}
 
+	// Bind is called under RTPSender.mu lock, call the RTPSender.GetParameters in goroutine to avoid deadlock
+	go func() {
```
Comment (cnderrauber) [inline pkg/sfu/downtrack.go:467]: revert the change since it is not the issue and we can get rtp header extension earlier

**Review by boks1971 — APPROVED** [review]
